livestatus/api/python/livestatus.py

```diff
@@ -498,7 +498,7 @@ def recv_response(self, query=None, add_headers="", timeout_at=None):
         try:
             # Headers are always ASCII encoded
             resp = self.receive_data(16)
```

Comment:
Would be better to use .decode("ascii") here.

Reply:
Updated the commit accordingly.

Reply:
@LarsMichelsen Ready to be merged now after the change?

Reply:
Not formally, since Travis isn't happy, but I'll fix it when I merge.